src/pages/dataSetsWip/Edit.tsx

```diff
@@ -106,7 +151,15 @@ export const useOnSubmitDataSetsEdit = (modelId: string) => {
                 })
             }
 
-            submitEdit(values, form, options)
+            const trimmedValues = {
```

Comment:
putting dataEntryForm in the omitted properties in dataSetValueFormatter didn't work for me. We need to set the association to the form from the data set, rather than vice versa. I didn't look into this a lot but just handled the omission within the body of the edit, but maybe this logic should be revisited